src/main/_data/sidebars/che_7_docs.yml

```diff
@@ -9,6 +9,7 @@ entries:
     url: che-7/introduction-to-eclipse-che
     output: web
   - title: Che quick-starts
+    url: che-7/che-quick-starts
```

Comment:
@themr0c, this doesn't work. Can't be used.
```suggestion
```

Reply:
Given the fact that src/main/pages/che-7/overview/assembly_che-quick-starts.adoc has the necessary for that page to be generated, it works. And it would be very useful to point the user to a landing page for the quick starts.

Reply:
@themr0c what I mean is that when you click this menu item in the navig sidebar, it won't open the page, even though it will be built. (Navig items that expand to reveal other items can't open their own page.)

Reply:
@themr0c OK, I see now that you changed the navig structure, and the 'Che quick starts' item no longer expand -- instead, it opens its own page. Please, don't change thing like that without consulting it first.

Reply:
OK.